lib/ansible/modules/windows/win_environment.ps1

```diff
@@ -43,44 +32,38 @@ $result = @{
     value = $value
 }
 
-if ($diff_support) {
-    $result.diff = @{}
-}
-
 if ($state -eq "present" -and $before_value -ne $value) {
+
     if (-not $check_mode) {
         [Environment]::SetEnvironmentVariable($name, $value, $level)
     }
     $result.changed = $true
 
-    if ($diff_support) {
+    if ($diff) {
         if ($before_value -eq $null) {
-            $result.diff.prepared = @"
-[$level]
-+$NAME = $value
-"@
+            $result.diff = @{
+                prepared = " [$level]`n+$NAME = $value`n"
```

Comment:
Should there be a space before the first `[`? Any reason for the capitalised `$NAME`?

Reply:
Yes, diff output shifts the line with a space.
The capitalized name is weird, it was there before :-) (lame excuse)